server/game/player.js

```diff
@@ -1045,7 +1045,7 @@ class Player extends Spectator {
 
         this.placeCardInPile({ card, location: targetLocation, bottom: options.bottom, wasFacedown: options.wasFacedown });
 
-        if(['dead pile', 'discard pile', 'revealed plots'].includes(targetLocation)) {
+        if(['dead pile', 'discard pile', 'revealed plots', 'out of game'].includes(targetLocation)) {
```

Comment:
What was the reason for needing `onCardPlaced` for out-of-game? (Seems safe enough, just not immediately clear)

Reply:
there would be no onCardPlaced event for putting a plot out of game with the new agenda